target/linux/ramips/dts/mt7621_edimax_ra21s.dts

```diff
@@ -0,0 +1,155 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "edimax,ra21s", "mediatek,mt7621-soc";
+	model = "Edimax RA21S";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,57600";
+	};
+
+	palmbus: palmbus@1E000000 {
+		i2c@900 {
+			status = "okay";
+		};
+	};
```

Comment:
Since you do not use the "palmbus:" label, you might as well remove it.
I'd personally prefer lower-case for the 1e000000 ...

Reply:
mt7621.dtsi already has i2c node and label. https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/dts/mt7621.dtsi#L102
So you can just use `&i2c { status = "okay"; };` outside of the root node.
BTW does this device really need i2c function?

Reply:
>  just use &i2c
done

>does this device really need i2c function?
No idea. How to check it?

Reply:
If you disable i2c and you can't find any malfunction (GPIO, network, RTC, etc), then maybe i2c can be safely removed.

Reply:
Have no access to the device right now. If i2c is critical, we have to wait until the end of the week to check changes.

Reply:
Note that device support PRs almost never are accepted fast. So, take your time to check everything and test properly.

Reply:
Do not see any problems after i2c disabling.